.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,239 @@
+name: Integration Tests
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - main
+      - develop
+      - release*
+  pull_request:
+    branches:
```

Comment:
You can just leave `pull_request:` empty and it'll run against all.

Reply:
I can leave it blank to run on all PRs, but I think that could be a bit of a waste of resources? Each test spins up at least a few different instances, like simulated chains, chainlink nodes, DBs, etc... Seems like it would be best to restrict that to more significant PRs, as long as those have a stable pattern in their naming scheme / branches they merge to.